www/source/docs/diagrams.html.md.erb

```diff
@@ -1,7 +1,25 @@
+---
+title: Diagrams
+---
+
+# Diagrams
 
-# <a name="infographics" id="infographics" data-magellan-target="infographics">Habitat Infographics</a>
 With Habitat comes a number of concepts - some you may be familiar with, some you may not. Not to worry, here you'll find a set of graphics that will help you and your team better understand the concepts and how they fit together into the larger Habitat ecosystem.
 
+## Table of Contents
+
+- [Habitat Architecture Overview](#habitat-architecture-overview)
```

Comment:
Let's drop the word `Habitat` at the beginning of all of these, and in their headings also.